### PR TITLE
add Jenkinsfile

### PR DESCRIPTION
**Pull request by steveblackmon**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  stages {
+    stage('Install dependencies') {
+      steps {
+        sh 'npm install'
+      }
+    }
+     
+    stage('Test') {
+      steps {
+         sh 'npm test'
```
Comment (waldenraines) [inline Jenkinsfile:15]: Do we also want to run linting as part of the jenkins job?

Comment (steveblackmon) [reply Jenkinsfile:15]: sure!  whats the command?

Comment (waldenraines) [reply Jenkinsfile:15]: `npm lint`

Comment (steveblackmon) [reply Jenkinsfile:19]: added

**Review by waldenraines — APPROVED** [review]

**Pull request merged**